Ginger/Ginger/UserControlsLib/TextEditor/Gherkin/GherkinPage.xaml.cs

```diff
@@ -514,6 +510,25 @@ private void GeneratBFButton_Click(object sender, RoutedEventArgs e)
         {
             if (!isBFexists)
             {
+                BusinessFlowsFolderTreeItem bfsFolder;
+                if (WorkSpace.Instance.BetaFeatures.BFUseSolutionRepositry)
+                {
+                    bfsFolder = new BusinessFlowsFolderTreeItem(WorkSpace.Instance.SolutionRepository.GetRepositoryItemRootFolder<BusinessFlow>(), eBusinessFlowsTreeViewMode.FoldersOnly);
+                }
+                else
+                {
+                    bfsFolder = new BusinessFlowsFolderTreeItem(eBusinessFlowsTreeViewMode.FoldersOnly);//create new tree each time for now to allow refresh
+                }                 
+                bfsFolder.Folder = GingerDicser.GetTermResValue(eTermResKey.BusinessFlows);
+                bfsFolder.Path = App.UserProfile.Solution.BusinessFlowsMainFolder;
+                bfsFolder.IsGingerDefualtFolder = true;
+                SingleItemTreeViewSelectionPage mTargetFolderSelectionPage = new SingleItemTreeViewSelectionPage(GingerDicser.GetTermResValue(eTermResKey.BusinessFlows), eImageType.BusinessFlow, bfsFolder, SingleItemTreeViewSelectionPage.eItemSelectionType.Folder, true, SingleItemTreeViewSelectionPage.eItemEnableEventType.Select);
```

Comment:
Consider that me and Yaron working now on modifing all tree items including BF's tree items to work with new SolutionRepository, that will break above code, if you will push code fist so later please check it is still working